lib/gstreamer/d3d12/decoder.py

```diff
@@ -0,0 +1,92 @@
+###
+### Copyright (C) 2024 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+import os
+import slash
+
+from ....lib import platform
+from ....lib.codecs import Codec
+from ....lib.common import get_media
+from ....lib.formats import PixelFormat
+from ....lib.gstreamer.decoderbase import BaseDecoderTest, Decoder as GstDecoder
+from ....lib.gstreamer.util import have_gst_element
+from ....lib.gstreamer.d3d12.util import mapformat, mapformatu
+
+class Decoder(GstDecoder):
+  format  = property(lambda s: mapformatu(super().format))
+  pformat = property(lambda s: mapformat(super().format))
+
+@slash.requires(*have_gst_element("d3d12"))
+class DecoderTest(BaseDecoderTest):
+  DecoderClass = Decoder
+
+  def before(self):
+    super().before()
```

Comment:
This is redundant.  This override can be removed since there is no extra stuff to do here.  The super class will automatically be called if it's not overridden.